### ISSUE TITLE
Initial Deployment

### ISSUE THREAD

**Issue by h36ahmed** [hayes0724/shopify-packer#38]
**Describe the bug**
I keep getting this error.

```
Section 1: settings is required, Section 1: 'theme_name' is not a valid attribute, Section 1: 'theme_version' is not a valid attribute, Section 1: 'theme_author' is not a valid attribute, Section 1: 'theme_documentation_url' is not a valid attribute, and Section 1: 'theme_support_email' is not a valid attribute
```

**To Reproduce**
packer theme:create --name=sample-theme-name
packer start

**Expected behavior**
There should be no errors.

**Screenshots**

**System (please complete the following information):**
 - OS: Mac OS Catalina
 - v10.21.0
 - 6.14.4
 - Chrome

**Packer (please complete the following information):**
 - Version 1.3.2

**Theme (please complete the following information):**
 - [Shopify Packer theme](https://github.com/hayes0724/packer-blank-theme) Yes

**Comment by mrpacman101** [comment]
@h36ahmed 
Open your /config/settings_schema.json

Change:
```
[
  {
    "name": "packer-blank-theme",
    "theme_name": "Packer blank theme",
    "theme_version": "1.0.7",
    "theme_author": "Eric Hayes",
    "theme_documentation_url": "https://github.com/hayes0724/packer-blank-theme",
    "theme_support_url": "https://github.com/hayes0724/packer-blank-theme/issues"
  }
]
```
to ->
```
[
  {
    "name": "theme_info",
    "theme_name": "Packer blank theme",
    "theme_version": "1.0.7",
    "theme_author": "Eric Hayes",
    "theme_documentation_url": "https://github.com/hayes0724/packer-blank-theme",
    "theme_support_url": "https://github.com/hayes0724/packer-blank-theme/issues"
  }
]
```

@hayes0724 
The issue in the blank theme is that it `"name": `must contain `"theme_info"` as value instead.

**Comment by h36ahmed** [comment]
Resolved. Thanks! Perhaps, update the repo or add some documentation for this. Will be really helpful for future! Thanks though for your hardwork and sharing this setup. Really like it.

**Comment by hayes0724** [comment]
Thanks for the feedback, this is fixed in packer theme [v1.0.9](https://github.com/hayes0724/packer-blank-theme/releases/tag/v1.0.9)
